assets/javascripts/discourse/connectors/category-custom-settings/solved-settings.hbs

```diff
@@ -12,4 +13,16 @@
       </label>
     </div>
   </section>
-{{/unless}}
\ No newline at end of file
+{{/unless}}
+
+<section class="field auto-close-solved-topics">
+  <label for="auto-close-solved-topics">
+    {{i18n "category.num_featured_topics"}}
```

Comment:
I think you might need a new label here?

Reply:
Oops, I pushed from the incomplete branch. Forced pushed the new branch. Thanks for catching this!